spanner/api/Spanner/GetBackups.cs

```diff
@@ -0,0 +1,107 @@
+
+            Action<List<Backup>> printBackups = backups =>
+            {
+                backups.ForEach(backup =>
+                {
+                    Console.WriteLine($"Backup Name : {backup.Name}");
+                });
+            };
+
+            // List all backups.
+            Console.WriteLine("All backups:");
+            var allBackups = databaseAdminClient.ListBackups(parentAsInstanceName).ToList();
+            printBackups(allBackups);
+
+            ListBackupsRequest request = new ListBackupsRequest
+            {
+                ParentAsInstanceName = parentAsInstanceName,
+            };
+
+            // List backups containing backup name.
+            Console.WriteLine($"Backups with backup name containing {backupId}:");
+            request.Filter = $"name:{backupId}";
+            var backupsWithName = databaseAdminClient.ListBackups(request).ToList();
+            printBackups(backupsWithName);
+
+            // List backups on a database containing name.
+            Console.WriteLine($"Backups with database name containing {databaseId}:");
+            request.Filter = $"database:{databaseId}";
+            var backupsWithDatabaseName = databaseAdminClient.ListBackups(request).ToList();
+            printBackups(backupsWithDatabaseName);
+
+            // List backups that expire within 30 days.
+            Console.WriteLine("Backups expiring within 30 days:");
+            string expireTime = DateTime.UtcNow.AddDays(30).ToString("O");
+            request.Filter = $"expire_time < \"{expireTime}\"";
+            var expiringBackups = databaseAdminClient.ListBackups(request).ToList();
+            printBackups(expiringBackups);
+
+            // List backups with a size greater than 100 bytes.
+            Console.WriteLine("Backups with size > 100 bytes:");
+            request.Filter = "size_bytes > 100";
+            var backupsWithSize = databaseAdminClient.ListBackups(request).ToList();
+            printBackups(backupsWithSize);
+
+            // List backups created in the last day that are ready.
+            Console.WriteLine("Backups created within last day that are ready:");
+            string createTime = DateTime.UtcNow.AddDays(-1).ToString("O");
+            request.Filter = $"create_time >= \"{createTime}\" AND state:READY";
+            var recentReadyBackups = databaseAdminClient.ListBackups(request).ToList();
+            printBackups(recentReadyBackups);
+
+            // List backups in pages.
+            Console.WriteLine("Backups in batches of 5:");
+            int pageSize = 5;
+            string nextPageToken = string.Empty;
+            do
+            {
+                var response = databaseAdminClient.ListBackups(parentAsInstanceName, nextPageToken);
+
+                Page<Backup> currentPage = response.ReadPage(pageSize);
+                foreach (Backup backup in currentPage)
+                {
+                    Console.WriteLine($"Backup Name : {backup.Name}");
```

Comment:
If instead of a list you define the method as PrinBackups(IEnumerable<Backup> backups)... you can call it here as well.